src/packages/core/src/options/server-options.ts

```diff
@@ -63,20 +63,24 @@ export const ServerOptions: Definitions<ServerConfig> = {
     normalize,
     shortDescription: "Enable a websocket server.",
     default: () => true,
-    legacyName: "ws"
+    legacyName: "ws",
+    cliType: "boolean",
+    cliAliases: []
   },
   wsBinary: {
     normalize,
     shortDescription:
       "Whether or not websockets should response with binary data (ArrayBuffers) or strings.",
-    default: () => "auto"
+    default: () => "auto",
+    cliChoices: [true, false, "auto"] as any[]
   },
   rpcEndpoint: {
     normalize,
     shortDescription:
       "Defines the endpoint route the HTTP and WebSocket servers will listen on.",
     default: () => "/",
-    defaultDescription: "'/rpc/v0' for Filecoin, '/' otherwise"
+    defaultDescription: "'/rpc/v0' for Filecoin, '/' otherwise",
```

Comment:
Sorry, I must have missed where this `rpcEndpoint` was added previously. I guess there isn't really a better place to put it? :-/

If not, let's update default to take the `config`. IIRC, it should have a `flavor` property on it, so we can actually generate the right default via https://github.com/trufflesuite/ganache-core/blob/next/src/packages/options/src/create.ts#L45

If it doesn't have a flavor property... UI guess we need to look into how hard it is to add it in.

Reply:
> Sorry, I must have missed where this rpcEndpoint was added previously.

And by no part of your fault! Apparently I missed making a PR where it's implemented that's supposed to be upstream of this branch. I think it was ambiguous until the `next` CLI implementation settled down since rebasing was an active target

> I guess there isn't really a better place to put it? :-/

I think this is the right place for it, but there is potential down the road that it would be moved out due to potential need to support more routes than just the main RPC endpoint. For now, I think this is a good place for it.

> If not, let's update default to take the config. IIRC, it should have a flavor property on it, so we can actually generate the right default via next/src/packages/options/src/create.ts#L45
> 
> If it doesn't have a flavor property... UI guess we need to look into how hard it is to add it in.

Ah! I'll look into this

Reply:
Ok, the work for this is now in PR #726, which includes the preface to adding `rpcEndpoint` and the correction to use `config.flavor` for the default